Week3/homework/giveCompliment.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+const giveCompliment = (name) => {
+    // Array with ten Complimen
+    const complimentArray = ['great', 'awesome', 'great', 'nice', 'Elegant', 'Elegant', 'Majestic', 'Lovely', 'Striking', 'Amazing'];
+    // to ronden the content of the array 
+    const randomArray = Math.floor(Math.random() * complimentArray.length);
```

Comment:
nit: `randomIndex` might be a clearer name

Reply:
Thank you for the feed back I changed the name